### PR TITLE
docs: Standardized scripts in request_files Update request_files script and README to match the standard like in other lambdas and create build_api.sh

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Update request_files script and README to match the standard like in other lambdas and create build_api.sh

Addresses [ORCA-434: Update request_files script and README to match the standard like in other lambdas and create build_api.sh](https://bugs.earthdata.nasa.gov/browse/ORCA-434)

## Changes

* Updated scripts
* Generated API
* Added `nosec` to some randomized test values

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## How Has This Been Tested?

Scripts run locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] User documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Unit tests
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: tasks/request_files/README.md [hunk 1]
```diff
@@ -28,304 +26,4 @@ ex> (podr) λ aws s3api head-object --bucket podaac-sndbx-cumulus-glacier --key
 ```
```
Comment (bhazuka) [inline tasks/request_files/README.md:26]: May want to add a sentence and link to the schemas like we do in the other Lambdas

**Review by bhazuka — CHANGES_REQUESTED** [review]
Overall looks good to me. I would do 1 minor addition to README. See my note.

**Review by rizbihassan — CHANGES_REQUESTED** [review]
Looks good overall. 
Should we add input/output schema examples like in https://github.com/nasa/cumulus-orca/tree/develop/tasks/request_status_for_granule#inputoutput-schemas-and-examples?

**Comment by AndrewDorn** [comment]
> Looks good overall. Should we add input/output schema examples like in https://github.com/nasa/cumulus-orca/tree/develop/tasks/request_status_for_granule#inputoutput-schemas-and-examples?

Added

**Review by rizbihassan — APPROVED** [review]

**Review by bhazuka — APPROVED** [review]
Looks good to me.

**Pull request merged**